src/util/pollWeb3.js

```diff
@@ -0,0 +1,38 @@
+import Web3 from 'web3'
```

Comment:
This all looks great @coostendorp!! Maybe a parallel impl (Haven't tried it yet) could be to use Drizzle for blockchain state caching for the vue front end.